### DISCUSSION TITLE
Access public/restricted in `npm publish` command is misleading

### DISCUSSION THREAD

**Discussion by y-nk** [npm/feedback#545] [Registry] [answered]
The cli command allow passing `--access` at `npm publish` time, which implicitly lead (me) to believe that _each_ publication could have a _different_ access policy.

This comes particularly handful when it comes to company related work, where a package "A" could be released publicly under the "latest" dist tag, but a "testing" dist tag could remain private.

I'd like to know if it's intentional or not ? how come that when i publish `--access restricted --tag test`, the whole package turns private ?

**Comment by MylesBorins** [comment, marked as answer]
@y-nk permissions are not granular per version. This is a core product feature that we would not be able to change at this time. That said there are potentially ways we could improve the documentation around this. Was there a particular place in the docs you found confusing?
